src/components/AuthImg.vue

```diff
@@ -33,6 +33,11 @@ const loadImage = async () => {
     const authCondition = props.authCondition ?? defaultAuthCondition
     const headers = authCondition(props.src) ? getAuthHeaders(getAuthenticationToken()) : {}
 
+    if (Object.keys(headers).length === 0) {
```

Comment:
This condition can be improved for readability. I would not be able to understand it if I didn't know the context. Can you use a variable as a flag whether something needs authentication or not and then use this flag in the condition.

```
const requiresAuth = authCondition(props.src)

if (!requiresAuth) {
  // do something
}
```